engine/cache/cache.go

```diff
@@ -68,7 +68,9 @@ type LockStore interface {
 
 type ScoredSetStore interface {
 	ScoredSetAdd(ctx context.Context, key string, value interface{}, score float64) error
+	ScoredAppend(ctx context.Context, key string, value interface{}) error
 	ScoredSetScan(ctx context.Context, key string, from, to float64, dest interface{}) error
+	SetCard(key string) (int, error)
```

Comment:
ScoredSetCard

Reply:
SetCard already exist, don't want to duplicate implementation